plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -0,0 +1,1490 @@
+	 *
+	 * @param WP_Comment|mixed $item Review or reply being rendered.
+	 * @return void
+	 */
+	protected function column_date( $item ) : void {
+
+		$submitted = sprintf(
+			/* translators: 1 - Product review date, 2: Product review time. */
+			__( '%1$s at %2$s', 'woocommerce' ),
+			/* translators: Review date format. See https://www.php.net/manual/datetime.format.php */
+			get_comment_date( __( 'Y/m/d', 'woocommerce' ), $item ),
+			/* translators: Review time format. See https://www.php.net/manual/datetime.format.php */
+			get_comment_date( __( 'g:i a', 'woocommerce' ), $item )
+		);
+
+		ob_start();
+
+		?>
+		<div class="submitted-on">
+			<?php
+
+			if ( 'approved' === wp_get_comment_status( $item ) && ! empty( $item->comment_post_ID ) ) :
+				printf(
+					'<a href="%1$s">%2$s</a>',
+					esc_url( get_comment_link( $item ) ),
+					esc_html( $submitted )
+				);
+			else :
+				echo esc_html( $submitted );
+			endif;
+
+			?>
+		</div>
+		<?php
+
+		echo $this->filter_column_output( 'date', ob_get_clean(), $item ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+
+	/**
+	 * Renders the product column.
+	 *
+	 * @see WP_Comments_List_Table::column_response() for consistency.
+	 *
+	 * @param WP_Comment|mixed $item Review or reply being rendered.
+	 * @return void
+	 */
+	protected function column_response( $item ) : void {
+		$product_post = get_post();
+
+		ob_start();
+
+		if ( $product_post ) :
+
+			?>
+			<div class="response-links">
+				<?php
+
+				if ( current_user_can( 'edit_product', $product_post->ID ) ) :
+					$post_link  = "<a href='" . esc_url( get_edit_post_link( $product_post->ID ) ) . "' class='comments-edit-item-link'>";
+					$post_link .= esc_html( get_the_title( $product_post->ID ) ) . '</a>';
+				else :
+					$post_link = esc_html( get_the_title( $product_post->ID ) );
+				endif;
+
+				echo $post_link; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+
+				$post_type_object = get_post_type_object( $product_post->post_type );
+
+				?>
+				<a href="<?php echo esc_url( get_permalink( $product_post->ID ) ); ?>" class="comments-view-item-link">
+					<?php echo esc_html( $post_type_object->labels->view_item ); ?>
+				</a>
+				<span class="post-com-count-wrapper post-com-count-<?php echo esc_attr( $product_post->ID ); ?>">
+					<?php $this->comments_bubble( $product_post->ID, get_pending_comments_num( $product_post->ID ) ); ?>
+				</span>
+			</div>
+			<?php
+
+		endif;
+
+		echo $this->filter_column_output( 'response', ob_get_clean(), $item ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+
+	/**
+	 * Renders the type column.
+	 *
+	 * @param WP_Comment|mixed $item Review or reply being rendered.
+	 * @return void
+	 */
+	protected function column_type( $item ) : void {
+
+		$type = 'review' === $item->comment_type
+			? '&#9734;&nbsp;' . __( 'Review', 'woocommerce' )
```

Comment:
More a question than anything else (also, this is a minor detail and not a blocker): is this RTL friendly?

Reply:
It renders as `<Star Symbol> Review` -- I don't believe it would bother which side it is even in a RTL language since it's not changing the meaning of the translatable word next to it. If you believe otherwise we could further wrap this with a `is_rtl()` conditional though.